apps/web/src/pages/get-started/consts/EchoUseCase.const.tsx

```diff
@@ -0,0 +1,95 @@
+import { StepDescription, StepText } from './shared';
+import { OnboardingUseCase } from './types';
+import { OnboardingUseCasesTabsEnum } from './OnboardingUseCasesTabsEnum';
+import { useSegment } from '@novu/shared-web';
+import { CodeSnippet } from '../components/CodeSnippet';
+import { css } from '../../../styled-system/css';
+
+const COMMAND = 'npx novu-labs@latest echo';
+
+const EchoCodeSnippet = () => {
+  const segment = useSegment();
+
+  return (
+    <CodeSnippet
+      command={COMMAND}
+      // value from designs
+      className={css({ maxW: '400px' })}
+      onClick={() => {
+        segment.track(`Copy echo command - [Get Started]`);
+      }}
+    />
+  );
+};
+
+const textClassName = css({
+  color: 'legacy.B60',
+  maxW: '400px',
+});
+
+export const EchoUseCaseConst: OnboardingUseCase = {
+  title: 'Echo notifications center',
+  type: OnboardingUseCasesTabsEnum.ECHO,
+  description:
+    'Novu Echo SDK allows you to write notification workflows in your codebase' +
+    ' locally right in your IDE as well as preview and edit the channel specific content in real-time.' +
+    ' You can use Echo with React Email, MJML, or any other template generator.',
+  useCaseLink: 'https://docs.novu.co/echo/quickstart',
+  steps: [
+    {
+      title: 'Configure endpoint',
+      Description: function () {
+        return (
+          <StepDescription>
+            <StepText>To get started, open your terminal and launch the Dev Studio.</StepText>
+            <EchoCodeSnippet />
+          </StepDescription>
+        );
+      },
+    },
+    {
+      title: 'Create a workflow',
+      Description: function () {
+        return (
+          <StepDescription>
+            <p className={textClassName}>
```

Comment:
```
const StepText = styled('p', text);
```
(`text` should be imported from `styled-system/recipes` -- you can search for examples)

```suggestion
            <StepText variant="secondary">
```
We can also use this to get built-in styles